layouts/partials/head/style.html

```diff
@@ -1,3 +1,11 @@
 {{ $sass := resources.Get "scss/style.scss" }}
 {{ $style := $sass | resources.ToCSS | minify }}
-<link rel="stylesheet" href="{{ $style.RelPermalink }}">
\ No newline at end of file
+<link rel="stylesheet" href="{{ $style.RelPermalink }}">
+
+{{ if .Site.Params.comments.enabled }}
+    {{ if eq .Site.Params.comments.provider "cactus" }}
+        {{ $sass_cactus := resources.Get "scss/partials/comments/provider/cactus.scss" }}
+        {{ $style_cactus := $sass_cactus | resources.ToCSS | minify }}
+        <link rel="stylesheet" href="{{ $style_cactus.RelPermalink }}">
+    {{ end }}
+{{ end }}
```

Comment:
These changes are no longer needed since the style is loaded inline in HTML file

Reply:
Whoops yeah mixed it up apparently

Reply:
Fixed in f4a9761867b11350fcf7bfefddf11afc60932e1a